addons/website/models/website_visitor.py

```diff
@@ -332,15 +333,25 @@ def _merge_visitor(self, target):
         self.website_track_ids.visitor_id = target.id
         self.unlink()
 
-    def _cron_unlink_old_visitors(self):
+    def _cron_unlink_old_visitors(self, auto_commit=True):
```

Comment:
Is it really worth adding this new param just to be able to make tests pass?
I'd go with the common way (seeing the code) of doing it, simple define in the method
```py
auto_commit = not getattr(threading.current_thread(), 'testing', False)
```
(less diff, no signature change even if stable friendly, no "useless" (IMHO) param)

Reply:
@odoo/rd-security Hello, the suggestion here is triggering the ci/security. Could you please have a look? See my previous message about rationale :+1: